File: 백준/Bronze/14487. 욱제는 효도쟁이야！！/욱제는 효도쟁이야！！.cc
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    int numOfTown;

    cin >> numOfTown;

    vector<int> moveCost;
    int totalCost = 0;

    for (int i = 0; i < numOfTown; i++) {
        int inputCost;

        cin >> inputCost;

        moveCost.push_back(inputCost);

        totalCost = totalCost + inputCost;
    }

    sort(moveCost.begin(), moveCost.end());

    totalCost = totalCost - moveCost.back();

    cout << totalCost;

    return 0;
}

File: 백준/Bronze/14487. 욱제는 효도쟁이야！！/욱제는 효도쟁이야！！.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int numOfTown;

        numOfTown = sc.nextInt();

        int[] moveCost = new int[numOfTown];
        int totalCost = 0;

        for (int i = 0; i < numOfTown; i++) {
            moveCost[i] = sc.nextInt();
            totalCost = totalCost + moveCost[i];
        }

        Arrays.sort(moveCost);

        totalCost = totalCost - moveCost[numOfTown - 1];

        System.out.print(totalCost);

        sc.close();
    }
}
